airbyte-integrations/bases/base-normalization/dbt-project-template/packages.yml

```diff
@@ -2,4 +2,4 @@
 
 packages:
   - git: "https://github.com/fishtown-analytics/dbt-utils.git"
-    revision: 0.6.2
+    revision: 0.6.4
```

Comment:
neat!

is this where we would add the community maintained MySQL DBT package to work with it?

Reply:
yes, exactly!